### DISCUSSION TITLE
Mis Prácticas

### DISCUSSION THREAD

**Discussion by 3leni** [Launch-X-Latam/MisionFrontEnd#417] [Show and tell]
Hola, espero se encuentren bien, me gustaría mucho mostrarles mis practicas durante esta misión. 
 Módulo 1:  https://github.com/3leni/Abogabot
Módulo 2 HTML:  
* Repositorio: https://github.com/3leni/Pasteleria-LUZZIE
* En web:  https://3leni.github.io/Pasteleria-LUZZIE/

Módulo 3 CSS:
*Repositorio: https://github.com/3leni/Vacunas
*En web:  https://3leni.github.io/Vacunas/

Módulo 4 Js:
*Repositorio:  https://github.com/3leni/Pokedex-Js
En web:  https://3leni.github.io/Pokedex-Js/

**Comment by jeb-repo** [comment]
muy lindo quedo. que programa usaste para diagrama de flujo?

**Reply by GonDev1** [reply]
Que tal Jeb-repo, puedes utilizar [](https://miro.com/es/) es muy intuitiva y puedes compartir tu trabajo con un link, espero te sea de ayuda, saludos ✌

**Comment by GonDev1** [comment]
Hola 3leni

Esta muy completo y ordenado tu repo sigue así, saludos ✌

**Comment by mitziyaz** [comment]
Hola! esta precioso, una duda, como haces para poder obtener el link de tu pagina html, veo que tienen la extensiòn github.io, a donde accedes para hacerlas??

**Reply by GonDev1** [reply]
Debes subir tu pagina a GitHub Pages 😉
